internal/vlan/retrieve.go

```diff
@@ -41,19 +41,28 @@ func (c *Client) Retrieve() *cobra.Command {
 
 		RunE: func(cmd *cobra.Command, args []string) error {
 			cmd.SilenceUsage = true
-			vnets, _, err := c.Service.List(projectID, c.Servicer.ListOptions(nil, nil))
+			request := c.Service.FindVirtualNetworks(cmd.Context(), projectID).Include(nil).Exclude(nil)
+			filters := c.Servicer.Filters()
+			if filters["facility"] != "" {
+				request = request.Facility(filters["facility"])
```

Comment:
Keeping the filter by facility functionality, if it still works, can't hurt - I think. Since it is in the API spec / SDK and the code path is already here, I agree with updating the as-is code.